feature-extensions/multiaccount/triggerJenkinsDeleteResources.py

```diff
@@ -56,4 +62,23 @@ def collectUserInput(args):
                       '(Empty will delete all): ') or "all"
 
 
+def jenkins_job_status(job_name, args):
+    try:
+        url = "http://%s:%s@%s/job/%s/lastBuild/api/json" \
+                % (args.jenkins_username, args.jenkins_password, args.jenkins_url, job_name)
+        data = requests.get(url).json()
+        if data['building']:
+            time.sleep(60)
+        else:
+            if data['result'] == "SUCCESS":
```

Comment:
Please change the messages, if we dont need we can remove them

Reply:
Removed.